kernel/smp.c

```diff
@@ -71,7 +71,7 @@ void z_smp_init(void)
 {
 	(void)atomic_clear(&start_flag);
 
-#if defined(CONFIG_SMP) && (CONFIG_MP_NUM_CPUS > 1)
+#if defined(CONFIG_SMP) && (CONFIG_MP_NUM_CPUS > 1) && !defined(CONFIG_SOF)
```

Comment:
lets not have CONFIG_SOF in here. Instead, I would define a new general Kconfig, for example CONFIG_SMP_START_ONE or whatever and make CONFIG_SOF select that...

Reply:
@nashif how about `CONFIG_SMP_BOOT_DELAY`?